docs/title-and-description.md

```diff
@@ -76,4 +76,55 @@ metadata:
 ```
 
 The above examples will render as rows like the below image:
-![More Markdown Examples](assets/workflow-title-and-description-markdown-complex.png)
+![More Markdown Examples](assets/title-and-description-markdown-complex-workflow.png)
+
+### Cluster Workflow Template
```

Comment:
```suggestion
### For `ClusterWorkflowTemplates`

> v3.6 and after

You can also add the `workflows.argoproj.io/title` and `workflows.argoproj.io/description` annotations with embedded markdown to a `ClusterWorkflowTemplate` to display in the list:
```

- Add the word "For" as a clarifier
  - In particular, headings pop up in docs searches, so plain "ClusterWorkflowTemplate" could be confusing.
  - I feel like there could be a better heading, but I'm not thinking of one off the top of my head right now
- You can use code backticks for resource names, which will cause it to be ignored by the spell checker.
  - It also is one word, so if you din't use backticks, you should add it to the [`.spelling`](https://github.com/argoproj/argo-workflows/blob/7286d499a60e38e226365ac0300571963379e00c/.spelling) file
    - (Note that CronWorkflow and WorkflowTemplate are already in that file, just not ClusterWorkflowTemplate)
- Add the version specifier, as I've mentioned before
- Introduce/describe the example, similar to the other existing sections

Reply:
Also can you move this ClusterWorkflowTemplate section to the bottom? It is the least used and generally should be after WorkflowTemplates as well